client/src/lib/dependency-injection/buildComponentContainer.js

```diff
@@ -23,18 +24,16 @@ const buildComponentContainer = (base = buildBaseContainer()) => ({
    * @param key
    * @param context
    * @param args
-   * @returns {XML}
+   * @returns {function}
    */
   get(key, context, ...args) {
-    const service = base.get.call(this, key, context, ...args);
+    const Component = base.get.call(this, key, context, ...args);
 
-    if (service.displayName && service.displayName.match(/\]$/)) {
-      return service;
-    }
+    const service = (props) => <Component identifier={context} {...props} />;
```

Comment:
The idea is to included the `context` as the `identifier` prop to the service found.
Straight forward and enables authors to tweak the identifier within the Component if they wanted (if that's even desired?)

Reply:
I worry about conflicts with a generic name like `identifier`. Maybe `injectorContext`? Seems more appropriate.

Reply:
yeah, could be something like that, cheers